src/runners/ingest_runner.py

```diff
@@ -15,6 +24,21 @@ def main():
         except Exception as e:
             log.error(f"failed to run {name}", e)
 
+    if connection_table:
+        tables = db.fetch(f"SHOW TABLES LIKE '{connection_table}' IN DATA")
+    else:
+        tables = db.fetch(CONNECTION_TABLE_QUERY)
+
+    for table in tables:
+        log.info(f"Starting {table['name']}")
+        options = yaml.load(table['comment'])
+        if 'source' in options:
```

Comment:
let's call this "module" or "module_name" instead of "source"

Reply:
![image](https://user-images.githubusercontent.com/42752788/58910680-0280f680-86cb-11e9-8770-019026050229.png)